standalone/standalone.go

```diff
@@ -118,6 +120,18 @@ func Handler(ch grpcdynamic.Channel, target string, methods []*desc.MethodDescri
 	})
 }
 
+func registerExamplesHandler(mux *http.ServeMux, uiOpts *handlerOptions) {
+	mux.HandleFunc("/examples", func(w http.ResponseWriter, r *http.Request) {
+		w.Header().Add("Content-Type", "application/json")
+		w.WriteHeader(200)
+		if len(uiOpts.examples) > 0 {
```

Comment:
nit: you could resolve this outside the call to `HandleFunc`, so the actual handler just blindly writes bytes.

Reply:
will work either way, and it's not really a performance issu